app/models/document.rb

```diff
@@ -225,8 +241,8 @@ class NotArchiveableError < StandardError; end
   )
   scope :publishable, -> { where(publishable: true) }
 
-  scope :for_publication, -> { active.publishable }
-  scope :for_display, -> { active.referenced_in_decision_notice }
+  scope :for_publication, -> { publishable }
+  scope :for_display, -> { referenced_in_decision_notice }
```

Comment:
Doesn't this change the behaviour? If there's a document where `publishable: true` and an archived date set, then previously `for_publication` would have excluded it but now would include it. But it depends on whether that could actually happen, of course.

Reply:
Yes, that's why where these scopes are used I've prefixed the call with `active` - doing it inside the scope affects how the `OR` is constructed elsewhere.